src/protocol/packets/s2c.rs

```diff
@@ -227,6 +227,14 @@ pub mod play {
         }
     }
 
+    def_struct! {
+        Commands {
+            count: VarInt,
```

Comment:
I think you should omit `count` field, as `Encode for Vec<T>` already encodes length.
https://github.com/valence-rs/valence/blob/c955ab8799ef23f27c9c222a4fff8743326682a3/src/protocol.rs#L599

Reply:
Oh, didn't notice. Good job